### PR TITLE
[RUM-6774] Update and expand Unity documentation

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What does this PR do? What is the motivation?

Unity is moving forward in public beta, so we want to expand on the current documentation offerings and bring it up to the level of other mobile SDKs

### Merge instructions
- [x] Please merge after reviewing

### CODE REVIEW

**Comment by jhgilbert** [comment]
Thanks for this update, @fuzzybinary! I've filed a [ticket](https://datadoghq.atlassian.net/browse/DOCS-9322) for the docs team to review it.

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/unity.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+`Telemetry Sample Rate`
+: Optional<br/>
+**Type**: Double<br/>
+**Default**: `20`
+The percentage rate at which Datadog sends internal telemetry data. A value of 100 means all telemetry data is sampled and sent to Datadog.
+
+### Automatic view tracking
+
+If you select `Enable Automatic Scene Tracking`, Datadog will hook into Unity's `SceneManager` to detect scenes loading and unloading, and start RUM Views appropriately. If you are using methods to move between scenes other than `SceneManager`, or would like to track changes in views that occur without `SceneManager`, you need to track views manually using `DdRum.StartView` and `DdRum.StopView`.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/unity.md:47]:
```suggestion
If you select `Enable Automatic Scene Tracking`, Datadog hooks into Unity's `SceneManager` to detect scenes loading and unloading, and start RUM Views appropriately. If you are using methods to move between scenes other than `SceneManager`, or would like to track changes in views that occur without `SceneManager`, you need to track views manually using `DdRum.StartView` and `DdRum.StopView`.
```

File: content/en/real_user_monitoring/error_tracking/mobile/unity.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+description: Learn how to track Unity errors with Error Tracking.
+aliases:
+- /real_user_monitoring/error_tracking/unity
+type: multi-code-lang
+code_lang: cs
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/mobile/unity.md:6]:
```suggestion
code_lang: unity
```

Needs to be "unity" so it appears in the tab name.

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/unity.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
+---
+title: RUM Unity Advanced Configuration
+description: Learn how to configure Unity Monitoring.
+code_lang: cs
```
Comment (rtrieu) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/unity.md:4]:
```suggestion
code_lang: unity
```

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/unity.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: RUM Unity Data Collected
+description: Learn about the data collected by Unity Monitoring.
+code_lang: cs
```
Comment (rtrieu) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/unity.md:4]:
```suggestion
code_lang: unity
```

File: content/en/real_user_monitoring/error_tracking/mobile/unity.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/rum/error-tracking
+[2]: https://app.datadoghq.com/rum/application/create
+[3]: /real_user_monitoring/mobile_and_tv_monitoring/setup/unity#setup
+[4]: https://www.npmjs.com/package/@datadog/datadog-ci
+[5]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/unity-symbols
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/mobile/unity.md:119]:
```suggestion
[5]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/unity-symbols
[6]: https://app.datadoghq.com/source-code/setup/rum
```

File: content/en/real_user_monitoring/error_tracking/mobile/unity.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+3. Execute code containing an error or crash. For example:
+
+   ```cs
+   void ThrowError() {
+    throw new Exception("My Exception")
+   }
+   ```
+
+4. For obfuscated error reports that do not result in a crash, you can verify symbolication and deobfuscation in [**Error Tracking**][8].
+5. For crashes, after the crash happens, restart your application and wait for the Unity SDK to upload the crash report in [**Error Tracking**][8].
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/mobile/unity.md:109]:
```suggestion
5. For crashes, after the crash happens, restart your application and wait for the Unity SDK to upload the crash report in [**Error Tracking**][1].
```

File: content/en/real_user_monitoring/error_tracking/mobile/unity.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+2. Run your application on a simulator, emulator, or a real device. If you are running on iOS, ensure that the debugger is not attached. Otherwise, Xcode captures the crash before the Datadog SDK does.
+3. Execute code containing an error or crash. For example:
+
+   ```cs
+   void ThrowError() {
+    throw new Exception("My Exception")
+   }
+   ```
+
+4. For obfuscated error reports that do not result in a crash, you can verify symbolication and deobfuscation in [**Error Tracking**][8].
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/mobile/unity.md:108]:
```suggestion
4. For obfuscated error reports that do not result in a crash, you can verify symbolication and deobfuscation in [**Error Tracking**][1].
```

File: content/en/real_user_monitoring/error_tracking/mobile/unity.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+datadog-ci unity-symbols upload --android
+```
+
+**Note**: Re-uploading a source map does not override the existing one if the build id has not changed.
+
+For a full list of options, see the `datadog-ci` [Unity Symbols documentation][5].
+
+### List uploaded symbol files
+
+See the [RUM Debug Symbols][10] page to view all uploaded symbols.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/mobile/unity.md:83]:
```suggestion
See the [RUM Debug Symbols][6] page to view all uploaded symbols.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Suggestions to fix some broken links (that were already present).

**Review by rtrieu — APPROVED** [review]

**Review by davidejones — APPROVED** [review]

**Pull request merged**